service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -49,7 +49,7 @@ public class PipelinesService {
   private final SamService samService;
 
   private static final String SEM_VER_REGEX_STRING =
-      "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$";
+      "^(0|[1-9]\\d*|[a-zA-Z_]*v\\d+)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$";
```

Comment:
since the "v" is so important, should we add a note about formatting somewhere, like maybe in openapi.yml at the wdlMethodVersion description?

Reply:
im not sure we want to list all the possible options in the openapi yaml, its not like `v` is required.  Shoudl we put the regex in the openapi yml description?

Reply:
i added the regex to the error message